new-log-viewer/src/services/decoders/JsonlDecoder/utils.ts

```diff
@@ -0,0 +1,92 @@
+import dayjs, {Dayjs} from "dayjs";
+import utc from "dayjs/plugin/utc";
+
+import {
+    JsonObject,
+    JsonValue,
+} from "../../../typings/js";
+import {
+    INVALID_TIMESTAMP_VALUE,
+    LOG_LEVEL,
+} from "../../../typings/logs";
+
+
+// eslint-disable-next-line import/no-named-as-default-member
+dayjs.extend(utc);
```

Comment:
The ESlint error is about that we are importing both the default import `dayjs` and `Dayjs (which is used as the default export)`. Can we change the import statement to this?
```
import dayjs from "dayjs";
```

Also, can we move this `dayjs.extend(utc);` to MainServiceWorker.ts ?